index6.js

```diff
@@ -0,0 +1,17 @@
+// Написать функцию которая принимает в себя строчное значение состоящее из одного слова. В слове могут быть буквы разных 
+// регистров. Результатом выполнения данной функции должна быть строка, в которой первая буквы заглавная, а все последующие 
+// в нижнем регистре. Если в слове четное количество букв, то и последняя буква должна быть заглавной.
+
+
+const capitalized = function (stringValue) {
+    const firstCapitalLetter = stringValue[0].toUpperCase();
+    const restchars = stringValue.slice(1).toLowerCase();
+
+    if (stringValue.length % 2 === 0) {
+        const lastCapitalLetter = stringValue[stringValue.length - 1].toUpperCase()
+        return firstCapitalLetter + restchars + lastCapitalLetter
```

Comment:
Seems that your function is working incorrectly. If numbers of chars is even, your function add the additional char to the end in an upper case.
To fix this issue, you have remove last char in restchars